blockchain/stake/internal/tickettreap/common.go

```diff
@@ -87,11 +88,27 @@ type Value struct {
 type treapNode struct {
 	key      Key
 	value    *Value
-	priority int
+	size     int // how many items are within this treap, including counting this node as 1
```

Comment:
The `nodeFieldsSize` need to be updated to properly reflect the changed here to the size so the calculations are accurate.  That also implies the tests will need to be updated.